### ISSUE TITLE
After updating server-settings.json by hand, the server won't start

### ISSUE THREAD

**Issue by audiodude** [m-chandler/factorio-spot-pricing#8]
I edited server-settings.json by hand over SSH. I believe I have messed something up because after I ran the docker restart command, docker ps doesn't show the container running anymore.

How do I manually relaunch the container, preferably with logs so I can see what I'm doing wrong?

**Comment by m-chandler** [comment]
https://hub.docker.com/r/factoriotools/factorio/

Have you tried:
`sudo docker logs ecs-factorio`

**Comment by audiodude** [comment]
I figured it out by externally validating my JSON, but then, because I didn't know the name of the container (apparently it's `ecs-factorio`) I had to restart the whole cloud formation in order to bring the factorio container back up.

My recommendation is a short section in the README, either specifically about debugging the server-settings and seeing the logs, or just in general how to stop, start, restart the container. I had to figure out how to do the latter on my own when adding my save file and I'm sure it's not the best way.

Thanks!
